app/src/ui-blokada/res/values/strings_widget.xml

```diff
@@ -7,9 +7,13 @@
     <string name="widget_show_counter">Display number of blocked requests</string>
 
     <!-- A configuration option of widget -->
```

Comment:
A comment here could have a bit more context explanation for the translators, on what is this option for.

Reply:
ok :)